ReactSkia/core_modules/RSkAlertManager.h

```diff
@@ -0,0 +1,65 @@
+/*
+* Copyright (C) 1994-2021 OpenTV, Inc. and Nagravision S.A.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+#pragma once
+
+#include "jsi/JSIDynamic.h"
+
+#include "core_modules/Alert.h"
+#include "include/core/SkFontMgr.h"
+#include "include/core/SkCanvas.h"
+#include "include/core/SkColor.h"
+#include "include/core/SkFont.h"
+#include "sdk/NotificationCenter.h"
+#include "sdk/RNSKeyCodeMapping.h"
+#include "sdk/WindowDelegator.h"
+#include "ReactCommon/TurboModule.h"
+#include "ReactSkia/JSITurboModuleManager.h"
+
+namespace facebook {
+namespace react {
+
+enum AlertWindowState {
+        ALERT_WINDOW_CREATE = 1, // Window is Created
+        ALERT_WINDOW_ACTIVE = 2, // Window is Active
+        ALERT_WINDOW_INACTIVE = 3, // Window is In-Active
+        ALERT_WINDOW_DESTRUCT = 4 // Window is Destructed
+    };
+
+class RSkAlertManager : public rns::sdk::WindowDelegator, public facebook::react::TurboModule {
+public:
+
+    RSkAlertManager(
```

Comment:
this class is already under facebook::react namespace , so do we need to use it during referencing